control-plane/roles/monitoring/templates/alertmanager_tls_secrets.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: alertmanager-client-tls
+  namespace: monitoring
+type: Opaque
+data:
+  client.crt: {{ zem_client_cert }}
```

Comment:
It's specific to fits. You can clean this up. Also all variables used by this role should be prefixed with the role name `monitoring_` in order to reduce the probability of overlapping variable names (convention is used by all other roles as well).

Reply:
Removed